src/components/Accordion/Accordion.stories.tsx

```diff
@@ -45,3 +46,21 @@ AccordionAsFAQSet.args = {
   children: faqContent,
   count: 3,
 };
+
+// One accordion with scrollable list
+export const AccordionScroll = () => (
+  <>
+    <div style={{ maxHeight: "300px", width: "500px" }}>
+      <Accordion
+        accordionLabel={"Content"}
+        modifiers={["fixed-height"]}
+        defaultOpen={true}
+      >
+        <ListStory {...ListStory.args}></ListStory>
+      </Accordion>
+    </div>
+    <div>Example content underneath</div>
+  </>
```

Comment:
Although I much prefer the use of fragment tags over meaningless DIVs, we might want to use a DIV for the time being otherwise storybook will put `<React.Fragment>` tags in your component's "View Code" area, which may be undesirable:   
![image](https://user-images.githubusercontent.com/6432596/110550422-ab128480-8101-11eb-9171-7f33a6ed913d.png)

Reply:
I fixed everything else, but I'm having trouble making the code block show up in the docs at all so I'll hold off on fixing this until I can get that to show up